test/stats/cli-flags/stats.test.js

```diff
@@ -30,10 +30,8 @@ describe('stats flag', () => {
     });
 
     it('should warn when an unknown flag stats value is passed', () => {
-        const { stderr, stdout } = run(__dirname, ['--stats', 'foo']);
+        const { stderr } = run(__dirname, ['--stats', 'foo']);
         expect(stderr).toBeTruthy();
         expect(stderr).toContain('* configuration.stats should be one of these:');
-        expect(stderr).toContain('"none" | "errors-only" | "minimal" | "normal" | "detailed" | "verbose" | "errors-warnings"');
-        expect(stdout).toBeTruthy();
```

Comment:
Why we remove this?

Reply:
new preset `summary` was added. but it will only be for webpack@next, So we will need to add more logic if we want to keep this assertion. 
```js
if (isWebpack5) {
  expect(stderr).toContain("none" | "summary" | "errors-only" | "errors-warnings" | "minimal" | "normal" | "detailed" | "verbose")
} else {
  expect(stderr).toContain("none" | "errors-only" | "errors-warnings" | "minimal" | "normal" | "detailed" | "verbose")
}
```

Reply:
@snitin315 Yes, let's do it :+1: